src/sig_stfl/lms/sig_stfl_lms_functions.c

```diff
@@ -469,6 +489,7 @@ OQS_STATUS oqs_deserialize_lms_key(OQS_SIG_STFL_SECRET_KEY *sk, const size_t sk_
 	memcpy(lms_sk, sk_buf, lms_sk_len);
 	lms_key_data->sec_key = lms_sk;
```

Comment:
It looks to me like `lms_key_data` is still set to null at this point, so this will probably cause a segfault.

Reply:
Thanks Spenser,  Memory allocated at line# 482. 

That section of code is ...

	lms_key_data = malloc(sizeof(oqs_lms_key_data));
	lms_sk = malloc(lms_sk_len * sizeof(uint8_t));

	if (lms_key_data == NULL || lms_sk == NULL) {
		goto err;
	}

	memcpy(lms_sk, sk_buf, lms_sk_len);
	lms_key_data->sec_key = lms_sk;
	lms_key_data->len_sec_key = lms_sk_len;
	lms_key_data->context = context;

Reply:
Whoops, not sure how I missed that. Apologies.